admin_manual/installation/example_openbsd.rst

```diff
@@ -0,0 +1,240 @@
+.. _openbsd_installation_label:
+
+Example installation on OpenBSD
+===============================
+
+In this install tutorial we will be deploying Nextcloud on a minimal OpenBSD with our own httpd(8), PHP, PostgreSQL and redis (for -stable or -current are the same steps).
+
+From a base installed OpenBSD system you can just do::
+
+    # pkg_add nextcloud
+    
+The extra packages::
+
+    # pkg_add postgresql-server redis pecl74-redis php-pdo_pgsql 
+
+
+This will take care of your dependencies and give you the options to choose which PHP version do you want.
+
+HTTPD(8)
+--------
+
+Create a virtualhost in ``/etc/httpd.conf`` and add the following content to it::
+
+    server "domain.tld" {
+        listen on egress tls port 443
+        hsts max-age 15768000
+
+	  tls {
+		    certificate "/etc/ssl/domain.tld_fullchain.pem"
+		    key "/etc/ssl/private/domain.tld_private.pem"
+	  }
+
+	  # Set max upload size to 513M (in bytes)
+	  connection max request body 537919488
+	  connection max requests 1000
+	  connection request timeout 3600
+	  connection timeout 3600
+
+	  block drop
+
+	  # Ensure that no '*.php*' files can be fetched from these directories
+	  location "/nextcloud/config/*" {
+		  block drop
+	  }
+
+	  location "/nextcloud/data/*" {
+		  block drop
+	  }
+
+	  # Note that this matches "*.php*" anywhere in the request path.
+	  location "/nextcloud/*.php*" {
+		  root "/nextcloud"
+		  request strip 1
+		  fastcgi socket "/run/php-fpm.sock"
+		  pass
+	  }
+
+	  location "/nextcloud/apps/*" {
+		  root "/nextcloud"
+		  request strip 1
+		  pass
+	  }
+
+	  location "/nextcloud/core/*" {
+		  root "/nextcloud"
+		  request strip 1
+		  pass
+	  }
+
+	  location "/nextcloud/settings/*" {
```

Comment:
This section is not longer necessary. Settings is nowadays an app in /nextcloud/apps.

Reply:
Cool, good to know.